docs/filter.rst

```diff
@@ -53,6 +53,7 @@ Selectors
     |                        | | geometry of the given type       |                        |
     |                        | | (point, line, polygon, or other) |                        |
     +------------------------+------------------------------------+------------------------+
+.. [#] keep in mind that osm ids are not unique between osm types. In order to exclude only a specific object the id needs to be combined with an osm type filter e.g. (not id:10905729 and type:way) 
```

Comment:
good suggestion to put this into a footnote.
but… couldn't one make the example you give even clearer if the `not` was omitted? because I could potentially see some confusion of beginners reading this and wondering why one has to put the `not` here if one just wants to select a single element with a specific id (as in the example in the table)? I'd use the following as an example: `type:node and id:1234`

Reply:
see 
dec25f3

Reply:
perfekt. that was my use case but the inverse is better for docs